packages/blocks/src/code-block/code-block.ts

```diff
@@ -287,6 +286,11 @@ export class CodeBlockComponent extends NonShadowLitElement {
       assertExists(vEditor);
       vEditor.requestUpdate();
     }
+
+    const richText = this.querySelector('rich-text');
+    assertExists(richText);
+    this._richTextResizeObserver.disconnect();
```

Comment:
I think you should also call `this._richTextResizeObserver.disconnect();` in `Lit.disconnect`